ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-release-x.y-presubmits.yaml

```diff
@@ -0,0 +1,1459 @@
+presubmits:
+  stackrox/stackrox:
+  - agent: kubernetes
+    always_run: true
```

Comment:
When I diff `stackrox-stackrox-release-4.0-presubmits.yaml` and this file, I see a bunch of differing `always_run:` values. Could you please adjust them to be consistent (either way)?